<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysiere C/C++ Quelldateien auf bedingte Kompilieranweisungen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Quellcode auswählen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Wähle das Quellcodeverzeichnis aus, in welchem sich die zu untersuchenden Dateien befinden.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="startdir">
               <property name="placeholderText">
                <string>Startverzeichnis ...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSearch">
               <property name="text">
                <string>Durchsuchen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="chkRecursiv">
             <property name="toolTip">
              <string>Wenn eine Datei von keinem der ausgewähltem Belange betroffen ist, dann diese im Ergebnis ignorieren</string>
             </property>
             <property name="text">
              <string>Rekursiv durchsuchen</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Dateitypen durch Komma getrennt angeben. Z.B.: Headerdateien (*.h), und Quellcodedateien (*.c,*.cpp)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFileFIlter"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAnalyse">
             <property name="text">
              <string>Analysieren</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Projektdatei auswählen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Analyseergebnisse können gespeichert werden. Nutze dieses Feld um ein gespeichertes Ergebnis wieder einzulesen</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineProjectFile">
               <property name="placeholderText">
                <string>Projektdatei</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSearchINI">
               <property name="text">
                <string>Durchsuchen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>104</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Analysierte Dateien:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblProcessedFiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSwitchToOutput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Zur Ausgabe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>#ifdef</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Gefundene bedingte Kompilieranweisungen</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="listConcerns">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Markierung:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QToolButton" name="btnToggle">
                    <property name="text">
                     <string>Invertieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnAll">
                    <property name="text">
                     <string>Alles</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnFilter">
                    <property name="toolTip">
                     <string>Öffnet ein Dialogfenster und fragt nach einem Begriff. Alle passende Einträge werden in der Liste zusätzlich ausgewählt.</string>
                    </property>
                    <property name="text">
                     <string>Begriff</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Markierte:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QToolButton" name="btnEnableSelected">
                    <property name="text">
                     <string>Aktivieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btndisableSelected">
                    <property name="text">
                     <string>Deaktivieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QToolButton" name="btnChangeColorSelected">
                    <property name="text">
                     <string>Farbe ändern</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Dateien</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QListWidget" name="listFiles">
                  <property name="selectionMode">
                   <enum>QAbstractItemView::MultiSelection</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Markierung:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QToolButton" name="btnToggle_2">
                    <property name="text">
                     <string>Invertieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnAll_2">
                    <property name="text">
                     <string>Alles</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnFilter_2">
                    <property name="toolTip">
                     <string>Öffnet ein Dialogfenster und fragt nach einem Begriff. Alle passende Einträge werden in der Liste zusätzlich ausgewählt.</string>
                    </property>
                    <property name="text">
                     <string>Begriff</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Markierte:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QToolButton" name="btnEnableSelected_2">
                    <property name="text">
                     <string>Aktivieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btndisableSelected_2">
                    <property name="text">
                     <string>Deaktivieren</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QToolButton" name="btnChangeColorSelected_2">
                    <property name="text">
                     <string>Farbe ändern</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Optionen</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="7" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Dateibalken:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinUmbruch">
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="chkUmbruch">
                  <property name="text">
                   <string>Umbruch:</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="13" column="1">
                 <widget class="QSpinBox" name="spinWidth">
                  <property name="minimum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>50</number>
                  </property>
                 </widget>
                </item>
                <item row="16" column="0">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>74</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="11" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Minimale Höhe</string>
                  </property>
                 </widget>
                </item>
                <item row="13" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Breite</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Maximale Höhe</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QSpinBox" name="spinMax">
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                  <property name="value">
                   <number>300</number>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QCheckBox" name="chkHideFiles">
                  <property name="text">
                   <string>Dateien ohne ausgewählte
#ifdef ausblenden</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="11" column="1">
                 <widget class="QSpinBox" name="spinMin">
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
                <item row="14" column="0" colspan="2">
                 <widget class="QCheckBox" name="chkVariableSize">
                  <property name="text">
                   <string>Dateibalken korrelieren
zur Dateigröße</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QCheckBox" name="chkVertical">
                  <property name="text">
                   <string>Vertikale Ausgabe</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QCheckBox" name="chkHideFiles2">
                  <property name="text">
                   <string>Datein ohne #ifdef
ausblenden</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QCheckBox" name="chkSortFilesSize">
                  <property name="text">
                   <string>Dateien nach Größe
sortieren</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRefresh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ausgabe neu erstellen</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWebView" name="webView">
           <property name="url">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btnOutputINI">
            <property name="text">
             <string>Ausgabe als Projektdatei</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutputPNG">
            <property name="text">
             <string>Ausgabe als PNG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutputPDF">
            <property name="text">
             <string>Ausgabe als PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutputSVG">
            <property name="text">
             <string>Ausgabe als SVG</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionINI_einlesen">
   <property name="text">
    <string>Laden ...</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_als">
   <property name="text">
    <string>Speichern als ...</string>
   </property>
  </action>
  <action name="actionGeneriere_SVG">
   <property name="text">
    <string>Exportieren als SVG ...</string>
   </property>
  </action>
  <action name="actionActionIFDEF">
   <property name="text">
    <string>actionIFDEF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
